app/views/expenditures/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="nav-header ">
+ <div>
+      <%= link_to  user_groups_path do %>
+        <%= image_tag "back.svg", style: 'height:10px; width:auto;' %>
+      <% end  %>
+    </div>
+  <h4 class="text-gray-4 ">Expenditures</h4> 
+  <div> </div>
+</div>
+
+<div class= "expenditures-total">
+  <div class="icon-container"> 
+    <%= image_tag "#{@group_id.icon}.png", style: 'height:40px; width:auto;' %> 
+  </div>
+<% if @expenditures.count > 0 %>
+  <div class=" flex-column">
+    <p> <strong>Category:</strong> <%= @group_id.name %> </p>
+    <p> <strong>Total spent:</strong> $<%= @expenditures.first.total_expenditures(@group_id) %></p>
+  </div>
+  <%end %>
+</div>
+
+<div id="expenditures">
+  <% @expenditures.each do |expenditure| %>
+    <%= render expenditure %>
+    <p>
+     
+    </p>
+  <% end %>
```

Comment:
- [ ] The list of payments should present ordered by the most recent. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/164910998-76e614bb-99d3-485f-ae60-403113f4cd08.png)